packages/design-tokens/docs/ColorTokens.stories.tsx

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable import/order */
+import React from "react"
+import { Box, Heading } from "@kaizen/component-library"
+import { Meta, Story } from "@storybook/react"
+import flatMap from "lodash.flatmap"
+import { useTheme } from "../react"
+import colorString from "color-string"
+
+export default {
+  title: "Design Tokens/Color Tokens",
+  parameters: {
+    layout: "fullscreen",
+    docs: {
+      description: {
+        component:
+          "This boarding is a representation of the design tokens available",
+      },
+    },
+  },
+} as Meta
```

Comment:
I'd probably avoid type casting if there's no need for it (since it can hide errors that TS normally picks up on)